design/baremetal-operator/bmc-events.md

```diff
@@ -136,7 +138,9 @@ implementations (if they exist at all).
 
 ## References
 
-- [Ironic Vendor Passthru for Subscriptions](https://storyboard.openstack.org/#!/story/2009061)
+<!-- cSpell:ignore Supermicro,DMTF -->
```

Comment:
Why not add these to the global excludes? They're commonly used.

Reply:
I probably added them, but I missed removing temporary ignores here.